build/ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -100,10 +100,10 @@
         non_unique: true
       loop:
         - { name: pmm, gid: 1000 }
+        - { name: pmm-agent, gid: 1001 }
```

Comment:
This was a dirty hack a year ago, which resulted in non-unique group and user ids.
In particular, `pmm-agent` overlaps with ssh_keys in RHEL9:

`
[root@pmm-server pmm] # ls -la /usr/local/percona/pmm2
total 48
drwxr-xr-x 1 pmm-agent ssh_keys 4096 Oct 31 08:40 .
drwxr-xr-x 1 pmm-agent ssh_keys 4096 Oct 31 08:37 ..
drwxr-xr-x 2 pmm-agent ssh_keys 4096 Oct 31 08:37 bin
drwxr-xr-x 1 pmm-agent ssh_keys 4096 Oct 31 08:38 cache
drwxr-xr-x 4 pmm-agent ssh_keys 4096 Oct 31 08:37 collectors
drwxr-xr-x 1 pmm-agent ssh_keys 4096 Oct 31 09:39 config
drwxr-xr-x 2 pmm-agent ssh_keys 4096 Oct 31 08:37 exporters
drwx------ 1 pmm-agent ssh-keys 4096 Oct 31 09:43 tmp
drwxr-xr-x 2 pmm-agent ssh_keys 4096 Oct 31 08:37 tools
`